### PR TITLE
Joey/add all urls

### PR DESCRIPTION
**Pull request by Joeyh021**
Added all of the URLs (and blank views for them), as written in the design doc.  Also did a few other housekeeping bits. Still a draft until people are happy with the structure of it. feel free to fuck with it
Feel free to pull this apart if any of it could be done better in django, especially the url dispatching with the `people` app.

### CODE REVIEW

File: events/urls.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 app_name = "events"
 
+# see views.py for a description of each URL
 urlpatterns = [
     path("", EventsIndexPage.as_view(), name="index"),
+    path("create", EventCreatePage.as_view()),
```
Comment (joshdavies14) [inline events/urls.py]: Commenting here but this applies anywhere. While this is fine and perfectly acceptable (this world work, there's nothing wrong with this), I'd recommend adding a slash at the end of each url, and giving it a name. It just makes it nicer to refer to in templates as you can type `{% url 'events:create' %} to point to the url or whatever, and the slash is just something I've already done, IDK if it actually makes a difference :P Very small things though lmao
```suggestion
    path("create/", EventCreatePage.as_view(), name="create"),
```

**Review by joshdavies14 — COMMENTED** [review]
Besides my one comment, I think this all LGTM - have you tested the URLs for the people app, OOI? It looks like it should work but you never know :P

**Comment by Joeyh021** [comment]
> Besides my one comment, I think this all LGTM - have you tested the URLs for the people app, OOI? It looks like it should work but you never know :P

Yeah, it works, just throws a template not found exception for now. When we have blank templates in we can add proper tests for all the url routing.

File: events/urls.py (hunk 1 above)
Comment (Joeyh021) [reply events/urls.py]: I’ll make all these changes in the morning, ty for pointing out

**Review by joshdavies14 — APPROVED** [review]

**Pull request merged**